src/share/classes/com/alibaba/rcm/internal/AbstractResourceContainer.java

```diff
@@ -62,6 +63,12 @@ public static AbstractResourceContainer current() {
         return SharedSecrets.getJavaLangAccess().getResourceContainer(Thread.currentThread());
     }
 
+    public abstract List<Long> getActiveTenantThreadIds();
+
+    public abstract Long getConsumedAmount(ResourceType resourceType);
+
+    public abstract Long getResourceLimit(ResourceType resourceType);
```

Comment:
TenantResourceContainer should implement this method, right?

Reply:
yes. 
![Uploading image.png…]()